portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +26,34 @@ function addRandomFact() {
   const factContainer = document.getElementById('fact-container');
   factContainer.innerText = fact;
 }
+
+
+async function getQuotes() {
+    const responseFromServer = await fetch('/welcome');
+    const quote = await responseFromServer.json();
+
+    
+
+    const word = quote[Math.floor(Math.random() * quote.length)];
+
+    const welcomeItems = document.getElementById('welcome-container');
+    welcomeItems.innerHTML = word;
+
+
+    // welcomeItems.appendChild(
```

Comment:
Do you want help with trying to make this approach work? If not, this commented code and the below function can be removed.